util/Migrator/DbMigrator.cs

```diff
@@ -24,6 +24,8 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
     }
 
     public bool MigrateMsSqlDatabaseWithRetries(bool enableLogging = true,
+        bool rerunable = false,
```

Comment:
🎨 Hate to do this since it's in so many places, but two Ns for "rerunnable". I suggest we go with a less obscure English word and use "repeatable".